src/lib/browse/layers/CrossingsLayerControl.svelte

```diff
@@ -0,0 +1,93 @@
+<script lang="ts">
+  import {
+    ColorLegend,
+    ExternalLink,
+    HelpButton,
+    InteractiveLayer,
+  } from "lib/common";
+  import { Checkbox } from "lib/govuk";
+  import { overwriteCircleLayer, overwritePmtilesSource } from "lib/maplibre";
+  import type { MapGeoJSONFeature } from "maplibre-gl";
+  import { map } from "stores";
+  import { colors } from "../colors";
+
+  const name = "crossings";
+  let show = false;
+
+  function tooltip(feature: MapGeoJSONFeature): string {
+    const crossingTypeString = feature.properties.crossing;
+    if (crossingTypeString === "no")
```

Comment:
Why is this special and not in the mapping below?  (and in case my other comment got lost, the style of using `if` without braces is different than everywhere else in the codebase. `prettier` doesn't change it right now, so just be aware it introduces inconsistency